gammapy/data/observations.py

```diff
@@ -92,8 +96,16 @@ def __init__(
         self._gti = gti
         self._events = events
         self._pointing = pointing
-        self._location = location
+        self._location = location  # this is part of the meta or is it data?
```

Comment:
For me, meta!

Reply:
OK. I tend to agree.
Currently there is a `location` argument in the `Observation` constructor (not mentioned in the docstring though). 
We can impose that `location` is passed in the helper `Observation.create` method and be stored in the `meta` there.

Opinions @bkhelifi , @maxnoe , @adonath ?